sintia/sintia.py

```diff
@@ -261,6 +261,27 @@ def is_rate_limited(self, action: str, user_id: int) -> bool:
             return await message.channel.send(f'No results found for `{argument}`')
 
         return await message.channel.send(search_result["link"])
+    
+    @command_handler('gif')
+    async def google_image_search(self, message: discord.Message , argument: str) -> None:
+        if not argument:
+            return
+
+        google_config = get_config_section('search.google')
+        results = await self.http_get_request('https://www.googleapis.com/customsearch/v1?' + urlencode({
+            'q': argument + " filetype:gif OR filetype:mp4 OR filetype:apng OR filetype:webp OR filetype:mng OR filetype:flif OR OR filetype:webm",
```

Comment:
As discussed, google will not accept `OR` searches of `filetype:`. 
In addition, I would try to find other `filetype:` in the search term and remoev them